tests/common.go

```diff
@@ -474,6 +474,13 @@ type PlatformCredentialStruct struct {
 	credUID  string
 }
 
+type (
+	// TestcaseOwner represents the owner of a Testcase
+	TestcaseOwner string
+	// TestcaseQuarter represents the fiscal quarter during which the Testcase is automated
+	TestcaseQuarter string
```

Comment:
Make it `Quarter`

Reply:
Just to be consistent with TestcaseOwner